README.rst

```diff
@@ -0,0 +1,183 @@
+
+The community.general collection
+  A special collection managed by the Ansible Community Team containing all the modules and plugins which shipped in Ansible 2.9 that don't have their own dedicated Collection. A work in progress can be found in `community.general <https://github.com/ansible-collection-migration/community.general/>`_ repository.
+
+Repository
+  The location of the source code included in a collection. Contributors make suggestions, fix bugs, and add features through the repository. Collection owners can host repositories on GitHub, Gerrit, or any other  source code repository platform they choose.
+ 
+ 
+Documentation
+==============
+
+* `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
+* `Using Developing Collections <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html>`_
+ 
+Work needed
+===========
+
+Ansible 2.9 already contains basic support for Collections.
+
+The majority of the Ansible 2.10 release cycle is for:
+
+* Defining what the split of collections should be
+* Defining which modules go into these new modules
+* Defining ansible-base (ie which modules stay in ansible/ansible)
+* Updating test infrastruture
+* Testing the changes
+* Getting feedack from *you*
+  
+
+We will soon begin the migration of content out of ansible/ansible, into its new component collection repositories.
+
+**Warning:** Dates subject to change
+
+* On 2nd March  2020, we will freeze the devel branch using protected branches, and we will create the temp-2.10-devel branch from devel. This date marks the end of merging non-base plugin/module PRs into ansible/ansible.
+
+* 9th March 2020, we will perform the initial migration against temp-2.10-devel, and we will do our initial testing of the components.
+
+* 23rd March 2020, we intend to unfreeze devel and merge temp-2.10-devel back into devel. From that point on, devel for ansible/ansible will be for the ansible-base project only.
+
+FAQ
+====
+
+Users of Ansible
+-----------------
+
+`Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
+
+Q: Once the next version of Ansible is released, will my playbooks still work
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For users of the community version of ansible pip/apt-get install ansible will continue to give you a working install of Ansible including the three thousand plus modules.
+
+Q: I deploy Ansible directly from devel. Is that still advised?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We recognize that many users install Ansible directly from devel, and generally we do our best to keep the devel branch relatively stable. These changes, however, will have a very large short-term impact, and we cannot guarantee that devel will be as stable as it has been in the past through this migration process.  
```

Comment:
s/relatively// (remove it) — already use 'generally' here, so no need for another adverb